app/controllers/application.rb

```diff
@@ -0,0 +1,4 @@
+get '/' do
+  @peeps = Peep.all
+  erb :index
```

Comment:
How would you go about creating a specific order for the peeps? The brief specifies chronological order.